### PR TITLE
Standard attachments

### PR DESCRIPTION
**Pull request by diegoreymendez**
This PR removes some of the custom attachment code we're using and tries to standardize on Apple's supported mechanism to handle attachments.

Replaces 451 lines of code with 206 - although the style and mechanisms can be improved a bit to be honest.

To test, just try out the attachment features we already had.

### CODE REVIEW

File: Aztec/Classes/Public/Extensions/NSAttributedString+Helpers.swift [hunk 1]
```diff
@@ -2,9 +2,9 @@ import Foundation
 import UIKit
 
 
-/// NSTextStorage Helpers
+/// NSAttributedString Extension for Attachments
 ///
-extension NSTextStorage
+extension NSAttributedString
```
Comment (jleandroperez) [inline Aztec/Classes/Public/Extensions/NSAttributedString+Helpers.swift]: Good call here!

File: Aztec/Classes/Public/TextKit/AztecTextAttachment.swift [hunk 2]
```diff
@@ -40,22 +35,80 @@ public class AztecTextAttachment: NSTextAttachment
         super.init(coder: aDecoder)
     }
 
+    // MARK: - Origin calculation
+
+    func xPosition(forContainerWidth containerWidth: CGFloat) -> Int {
+
```
Comment (jleandroperez) [inline Aztec/Classes/Public/TextKit/AztecTextAttachment.swift]: Extra spacing?

File: Aztec/Classes/Public/TextKit/AztecTextStorage.swift [hunk 3]
```diff
@@ -245,7 +261,7 @@ public class AztecTextStorage: NSTextStorage {
         return html
     }
 
-    public func setHTML(html: String, withDefaultFontDescriptor defaultFontDescriptor: UIFontDescriptor) {
+    func setHTML(html: String, withDefaultFontDescriptor defaultFontDescriptor: UIFontDescriptor) {
```
Comment (jleandroperez) [inline Aztec/Classes/Public/TextKit/AztecTextStorage.swift]: Shouldn't this one remain public?

File: Aztec/Classes/Public/TextKit/TextView.swift [hunk 4]
```diff
@@ -45,18 +40,6 @@ public class TextView: UITextView {
...
-    override public var bounds: CGRect {
-        didSet {
-            guard oldValue.size != bounds.size else {
-                return
-            }
-
-            attachmentManager.resizeAttachments()
-        }
-    }
-
```
Comment (jleandroperez) [inline Aztec/Classes/Public/TextKit/TextView.swift]: Super to see this go!

File: Aztec/Classes/Public/TextKit/AztecTextAttachment.swift (hunk 2 above)
Comment (jleandroperez) [inline Aztec/Classes/Public/TextKit/AztecTextAttachment.swift]: I've noticed we've got this extra spacing ... all over. Is this something we want / our standard, or it's just by chance?

Comment (diegoreymendez) [inline Aztec/Classes/Public/TextKit/AztecTextAttachment.swift]: As far as I know we don't have standards on spacing, which IMHO is kind of a good thing as it means we can use it to increase readability.

File: Aztec/Classes/Public/TextKit/AztecTextStorage.swift (hunk 3 above)
Comment (diegoreymendez) [inline Aztec/Classes/Public/TextKit/AztecTextStorage.swift]: To be honest I don't think it should.  Our text storage is not designed to be used directly, but only through `Aztec.TextView`, which has the necessary interface for setting the HTML.

File: Aztec/Classes/Public/Extensions/NSAttributedString+Helpers.swift (hunk 1 above)
Comment (diegoreymendez) [inline Aztec/Classes/Public/Extensions/NSAttributedString+Helpers.swift]: Ty!

File: Aztec/Classes/Public/TextKit/AztecTextAttachment.swift (hunk 2 above)
Comment (diegoreymendez) [inline Aztec/Classes/Public/TextKit/AztecTextAttachment.swift]: That said, I'll remove it from this particular method, since its not really needed.

**Comment by diegoreymendez** [comment]
Ty @jleandroperez!

**Pull request merged**